experimental/framework/fluid-static/src/rootDataObject.ts

```diff
@@ -106,23 +68,6 @@ export class RootDataObject
         await Promise.all(loadInitialObjectsP);
     }
 
-    public dispose() {
-        // remove our listeners and continue disposing
-        this.runtime.off("connected", this.connectedHandler);
-        this.runtime.off("disconnected", this.disconnectedHandler);
-        // After super.dispose(), all event listeners are removed so we need to emit first.
-        this.emit("dispose");
-        super.dispose();
-    }
-
-    public get audience(): IAudience {
-        return this.context.getAudience();
-    }
-
-    public get clientId() {
-        return this.context.clientId;
-    }
```

Comment:
Should `audience` and `clientId` members be provided on `IFluidContainer`, or is this an intentional removal?  It looks like this is removing the functionality but not adding it elsewhere

Reply:
Intentional remove because this is going to get moved to the service via @sumedhb1995 s work

Reply:
Yep, this will become available through the containerServices.

Although we may want to keep it here for a version and mark it as deprecated first so that anyone consuming it knows that they will have to switch over?